src/main/java/ch/wisv/areafiftylan/service/TicketServiceImpl.java

```diff
@@ -132,6 +132,13 @@ public void cancelTicketTransfer(String token){
         tttRepository.save(ttt);
     }
 
+    @Override
+    public Collection<TicketTransferToken> getTicketTransferTokensByUser(String username) {
```

Comment:
I'd also like to add that this query might have some problems. If the ticket is transferred the TicketOwnerUsername changes to the receiver. In this case the TicketOwnerUsername and the Token User would be the same. 

Short version: the underlying data changes once the transfer happens, and so the result of this query changes.